addons/prometheus/0.59.1-40.1.0/operator/matchlabel-release.yaml

```diff
@@ -0,0 +1,2 @@
+- op: remove
+  path: /spec/selector/matchLabels/release
```

Comment:
For my knowledge, why don't we yank this selector directly from inside `default.yaml` ?

Reply:
What do you mean?

Reply:
Yeah, that was my question :-D Let's resolve this comment.